corehq/apps/reports/standard/monitoring.py

```diff
@@ -1862,10 +1836,7 @@ def _total_row(self, rows, report_data, users):
 
     @property
     def get_all_rows(self):
-        if toggles.EMWF_WORKER_ACTIVITY_REPORT.enabled(self.request.domain):
-            return self.user_rows()
-        else:
-            return self.rows
```

Comment:
I expect there's a lot more that can be removed along with this - like the implementation of `rows` and anything that lives behind the `view_by_groups` check